src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp

```diff
@@ -862,6 +900,21 @@ void FixedwingPositionControl::navigation_capabilities_publish()
 	}
 }
 
+void FixedwingPositionControl::get_waypoint_heading_distance(float heading, float distance,
+					struct position_setpoint_s &waypoint_prev, struct position_setpoint_s &waypoint_next)
+{
+	/* XXX naive test implementation, cross-check */
+	waypoint_prev.valid = true;
+	waypoint_prev.lat = _global_pos.lat;
```

Comment:
@LorenzMeier Mark mentioned cross-track error yesterday in the dev call. I wasn't sure about that. Here you always take the current position for the new L1 track, so it will actually drift and doesn't compensate cross-track error?